properties/base.py

```diff
@@ -155,10 +155,15 @@ def __init__(self, **kwargs):
                                'or attribute'.format(key))
             setattr(self, key, kwargs[key])
 
-    def _get(self, name):
+    def _get(self, name, default=None):
         if name in self._backend:
             return self._backend[name]
 
+        if default is utils.undefined:
+            return None
+        if default is not None:
+            return default
+
```

Comment:
Yes, you are correct. This was intended as a quick fix for some problems other users were having with backwards incompatibility. However, they have already modified their code accordingly, so I don't think this PR will be merged.